### PR TITLE
feat: replace allowBigTransactions function with increaseTransactionL…

### PR DESCRIPTION
**Pull request by KristianBalaj**
The function `allowBigTransactions` was too limiting with the fixed setting that was provided by `allowBigTransactions`. This change adds the possibility to increase the limits even further.

Changing the base branch to main so closing this PR https://github.com/input-output-hk/plutus-apps/pull/605 that was created from `next-node` branch.

### CODE REVIEW

**Comment by KristianBalaj** [comment]
@sjoerdvisscher are those 3 still running tests supposed to be running that long (5 hours already)? Or it is just waiting for the approval?

**Review by sjoerdvisscher — APPROVED** [review]
Thank you for only applying `increaseTransactionLimits` as often as necessary in the use cases, that's actually really interesting info!

File: plutus-use-cases/test/Spec/Uniswap.hs
Comment (KristianBalaj) [reply plutus-use-cases/test/Spec/Uniswap.hs]: Updated 🙌

**Pull request merged**
